examples/source/async-source/example.py

```diff
@@ -32,13 +34,13 @@ async def read_handler(self, datum: ReadRequest) -> AsyncIterable[Message]:
         for each read request we process num_records and increment the read_idx to indicate that
         the message has been read and the same is added to the ack set
         """
-        if self.to_ack_set:
+        if len(self.to_ack_set) > 0:
```

Comment:
Any specific reason to change this?

Reply:
Just to make it more readable and direct

Reply:
Thanks for this! Updated it :)